yaksh/templates/yaksh/monitor.html

```diff
@@ -68,24 +68,39 @@
         {% if quiz %}
         {% if papers %}
         <div class="row">
-        <div class="card col-md-4">
-            <div class="row">
-                <div class="col"><b>Course Name:<br>Quiz Name:<br>Number of papers:<br>Papers completed:<br>Papers in progress:</b>
-                </div>
-                <div class="col">                
-                     {{ course.name }}<br>
-                     {{ quiz.description }}<br>
-                    {{ papers|length }}<br>
-                    {% completed papers as completed_papers %}
-                     {# template tag used to get the count of completed papers #}
-                      <b class="yakshred"> {{ completed_papers }} </b><br>
-                    {% inprogress papers as inprogress_papers %}
-                     {# template tag used to get the count of inprogress papers #}
-                     <b class="yakshred">  {{ inprogress_papers }} </b>
-                </div>
+        <div class="card col-md-10">
```

Comment:
This border does not align with the result table below.
The result table is slightly shifted towards the right.
Also these two tables are touching each other which doesn't look good.

Reply:
> Also these two tables are touching each other which doesn't look good.

Can you please clear you chache and check once again. Also try this with the updated PR